.doks/doks-cluster/Makefile

```diff
@@ -29,25 +31,24 @@ setup_vars:
 	@echo "docker_password = [REDACTED]"
 	@echo "docker_password = \"$(DOCKER_PASSWORD)\"" >> $(VARS_FILE)
 
-	echo "k8s_version = \"$(DO_K8S_SLUG)\"" >> $(VARS_FILE)
-	echo "node_type = \"$(NODE_TYPE)\"" >> $(VARS_FILE)
-	echo "num_nodes = $(NUM_NODES)" >> $(VARS_FILE)
-	echo "cluster_name = \"$(DOKS_CLUSTER_NAME)\"">> $(VARS_FILE)
+	@echo "k8s_version = \"$(DO_K8S_SLUG)\"" >> $(VARS_FILE)
+	@echo "node_type = \"$(NODE_TYPE)\"" >> $(VARS_FILE)
+	@echo "num_nodes = $(NUM_NODES)" >> $(VARS_FILE)
+	@echo "cluster_name = \"$(DOKS_CLUSTER_NAME)\"">> $(VARS_FILE)
+	@echo "main_namespace = \"$(NAMESPACE)\"">> $(VARS_FILE)
 
 .PHONY: set_cluster_context
 set_cluster_context:
 	echo '1' | doctl kubernetes cluster kubeconfig save ${DOKS_CLUSTER_NAME} --set-current-context
 
 .PHONY: start_cluster
-start_cluster: set_cluster_context helm_init_upgrade
+start_cluster: set_cluster_context cluster helm_init_upgrade
 
 .PHONY: cluster
 cluster: setup_vars create_terraformrc
 	echo '1' | $(TF) init -input=false -reconfigure
 	$(TF) workspace select $(WORKSPACE)
 	$(TF) apply -parallelism=$(TF_PARALLELISM) -var-file=$(VARS_FILE) -input=false --auto-approve
-	$(shell doctl kubernetes cluster kubeconfig save $(DOKS_CLUSTER_NAME) --set-current-context)
```

Comment:
We no longer provision helm through terraform so we no longer need this step nor the following terraform apply